lib/commands/file-movement.js

```diff
@@ -72,14 +72,24 @@ function verifyIsSubPath(originalPath, root) {
  *
  * @param {string} udid
  * @param {string} [bundleId]
+ * @param {string} [containerType]
  * @returns {Promise<any>}
  */
-async function createAfcClient(udid, bundleId) {
+async function createAfcClient(udid, bundleId, containerType) {
   if (!bundleId) {
     return await services.startAfcService(udid);
   }
   const service = await services.startHouseArrestService(udid);
-  return await service.vendContainer(bundleId);
+
+  const {
+    skipDocumentsContainerCheckInFileManagement = false,
+  } = await this.settings.getSettings();
+
+  return skipDocumentsContainerCheckInFileManagement
```

Comment:
Avoid using nested ternary operators. They are hard to parse. Vanilla if condition would be just fine